web/index.html

```diff
@@ -376,32 +437,36 @@ <h3>Used in</h3>
             return queryParams;
         }
 
-        // Decode base64 value mod 1000
-        function getChunk(value) {
-            let base64 = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-='
-            let result = 0
-            for (let i = 0; i < value.length; i++) {
-                result = (result * 64 + base64.indexOf(value[i]))
-            }
-            return Math.floor(result / 100)
-        }
-
-        const DATA_PREFIX = 'data/';
+        const PATH_PREFIX = window.location.host == 'expitau.github.io' ? '/' : '';
+        const CHUNK_WORKER_PATH = `${PATH_PREFIX}loadChunkWorker.js`;
+        const DATA_PATH = `${PATH_PREFIX}data/`;
+        const HELPER_PATH = window.location.host == 'expitau.github.io' ? 'https://expitau.github.io/InfiniteCraftWiki/helper'
+            : 'http://127.0.0.1:5501/web/helper.html';
 
         // Vue app
         const app = new Vue({
             el: '#app',
             data: {
                 data: { index: {}, list: [], costs: {}, craftingCache: {}, data: {}, metadata: { recipeCount: 0, loadedChunks: [] } },
+                chunkLoadState: [],
+                craftingTreeData: null,
+                bookmarkletData: "",
```

Comment:
This is used to allow the bookmarklet code to be fetched, then updated using a replace command to get the proper urls in place, and then inserted into the link whenever it is ready

Reply:
Since this is used to insert the value into the vue html for the link, then the only way to do that without it breaking as the virtual dom updated would be to have a bunch of nasty methods that run in javascript that find the specific element by id and replace it's link destination whenever it exists. That's so much worse than just having this variable also be managed by vue.

Reply:
I ended up turning it into a computed function instead because that is much cleaner to insert into the html dynamically